### ISSUE TITLE
Won't compile on Ubuntu 18.04.6 LTS  - Cran version

### ISSUE THREAD

**Issue by dpascall** [aadler/Delaporte#3]
Hi,
The Cran version doesn't seem to compile on Ubuntu 18.04.6 LTS. It throws this error.
```
delaporte.f90:356:39:

                         obsv(i) = real(minloc(svec, dim = 1, &
                                       1
Error: Can't find keyword named ‘back’ in call to ‘minloc’ at (1)
```
The archived 8.0.3 version was the latest I could get to compile properly.
Thanks!

**Comment by aadler** [comment]
Hello, @dpascall.

Per [GCC](https://gcc.gnu.org/onlinedocs/gfortran/MINLOC.html), "The BACK argument is available in Fortran 2008". Technically, the package requires Fortran 2008, but Professor Ripley suggested that `-std=f2008 ` be removed for compliance with more compilers, given that the major flavors of R all should have Fortran 2008. Are you saying that Ubuntu 18.04.6 LTS has an over 15 year old version of Fortran?

**Comment by aadler** [comment]
@dpascall, technically, I can remove that keyword since it has the default value of `FALSE`. However, before I do so, there is another Fortran 2008 dependency, and that is the `log_gamma` intrinsic. If you could please do me the favor of downloading a source package version of 8.2.0, and change lines 356--357 to look like:

```Fortran
obsv(i) = real(minloc(svec, dim = 1, &
                            mask = svec >= p(i)) - 1)
```

Then try building from source and see if Ubuntu is happy or still complains about missing `log_gamma`. If it is happy, I'll release a patch version without "back". Regardless, it may be valuable to use a version of Fortran somewhat less aged 😄 . Thanks!

**Comment by aadler** [comment]
@dpascall Were you able to check the above? I'd rather not push a change that doesn't work, so if you can't test this I may close the issue.

**Comment by dpascall** [comment]
Sorry. Only just got to this. With those changes it complied fine.

**Comment by aadler** [comment]
OK, if that change worked and you didn't get an issue with `log_gamma` then I can pull out the express reference to "BACK" in the next release since it uses the default `FALSE` anyway. Thank you.

**Comment by dpascall** [comment]
Great. Thanks very much for this!

**Comment by aadler** [comment]
Addressed in release [8.3.0](https://github.com/aadler/Delaporte/releases/tag/v8.3.0).
